### PR TITLE
Typo "Teams Room device"→"Teams Rooms device"

### PR DESCRIPTION
**Pull request by hyoshioka0128**
https://learn.microsoft.com/en-us/microsoftteams/devices/use-teams-panels#pair-a-teams-panel-with-a-microsoft-teams-room-on-android 
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/devices/use-teams-panels.md 
#PingMSFTDocs

### CODE REVIEW

File: Teams/devices/use-teams-panels.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ The right tile on the Home screen now appears in purple and displays the **Reser
     :::image type="content" source="../media/panels-right-tile-reserved-adhoc.png" alt-text="Home screen showing that meeting space is reserved for an ad hoc meeting.":::
 
     > [!NOTE]
-    > If the meeting space is a Microsoft Teams Room, you can _join_ this Teams meeting with the in-room Microsoft Teams Room or Surface Hub devices.
+    > If the meeting space is a Microsoft Teams Room, you can _join_ this Teams meeting with the in-room Microsoft Teams Rooms or Surface Hub devices.
```
Comment (JohanFreelancer9) [inline Teams/devices/use-teams-panels.md:123]:
```suggestion
    > If the meeting space is Microsoft Teams Rooms, you can _join_ this Teams meeting with the in-room Microsoft Teams Rooms or Surface Hub devices.
```

File: Teams/devices/use-teams-panels.md [hunk 2]
```diff
@@ -181,9 +181,9 @@ To access the **Teams admin settings** option:
 > [!NOTE]
 > Some OEMs may replace the **Teams admin settings** option with their own custom option. If you don't see the **Teams admin settings** option, check your device documentation for instructions on how to access your panel's admin settings.
 
-#### Pair a Teams panel with a Microsoft Teams Room on Android
+#### Pair a Teams panel with a Microsoft Teams Rooms on Android
```
Comment (JohanFreelancer9) [inline Teams/devices/use-teams-panels.md:184]:
```suggestion
#### Pair a Teams panel with Microsoft Teams Rooms on Android
```

File: Teams/devices/use-teams-panels.md [hunk 3]
```diff
@@ -195,7 +195,7 @@ On the Teams panel, sign in using your admin credentials.
 
 Check-in and room release settings let users check in to a meeting on Teams Panels at the room they reserved at the start of the meeting. If a user doesn’t check-in within a set amount of time after the meeting start time, the room is released and becomes available for others to reserve.
 
-When Teams Panels is paired with a Microsoft Teams Room on Android, check-in notifications can be enabled to appear on the front-of-room display when meetings run late.
+When Teams Panels is paired with a Microsoft Teams Rooms on Android, check-in notifications can be enabled to appear on the front-of-room display when meetings run late.
```
Comment (JohanFreelancer9) [inline Teams/devices/use-teams-panels.md:198]:
```suggestion
When Teams Panels is paired with Microsoft Teams Rooms on Android, check-in notifications can be enabled to appear on the front-of-room display when meetings run late.
```

File: Teams/devices/use-teams-panels.md [hunk 4]
```diff
@@ -222,11 +222,11 @@ This feature is disabled by default, but it can be enabled per device. To enable
 
 #### Enable or disable room capacity warnings
 
-Teams panels that are paired with a Teams Room on Android can display a warning message when a room is at or over capacity. To use this feature, the Teams Room must have a camera that supports people counting. Teams Rooms on Android support room capacity warnings without a Teams panel.
+Teams panels that are paired with a Teams Rooms on Android can display a warning message when a room is at or over capacity. To use this feature, the Teams Rooms must have a camera that supports people counting. Teams Rooms on Android support room capacity warnings without a Teams panel.
```
Comment (JohanFreelancer9) [inline Teams/devices/use-teams-panels.md:225]:
```suggestion
Teams panels that are paired with Teams Rooms on Android can display a warning message when a room is at or over capacity. To use this feature, Teams Rooms must have a camera that supports people counting. Teams Rooms on Android support room capacity warnings without a Teams panel.
```

File: Teams/devices/use-teams-panels.md [hunk 5]
```diff
@@ -222,11 +222,11 @@ This feature is disabled by default, but it can be enabled per device. To enable
...
 
 Room capacity warnings are disabled by default, but they can be enabled per device. To enable the feature, do the following:
 
-1. Follow the steps in [Pair a Teams panel with a Microsoft Teams Room on Android](#pair-a-teams-panel-with-a-microsoft-teams-room-on-android). The panel and the Teams Room device must be signed into the same resource account.
+1. Follow the steps in [Pair a Teams panel with a Microsoft Teams Rooms on Android](#pair-a-teams-panel-with-a-microsoft-teams-room-on-android). The panel and the Teams Rooms device must be signed into the same resource account.
```
Comment (JohanFreelancer9) [inline Teams/devices/use-teams-panels.md:229]:
```suggestion
1. Follow the steps in [Pair a Teams panel with Microsoft Teams Rooms on Android](#pair-a-teams-panel-with-a-microsoft-teams-room-on-android). The panel and the Teams Rooms device must be signed into the same resource account.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @hyoshioka0128, kindly commit the suggested changes. Many thanks.

**Comment by hyoshioka0128** [comment]
@JohanFreelancer9 - Thanks  for the suggestion.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@dstrome 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@dstrome
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request closed without merging**